OSVRUnreal/Plugins/OSVR/Source/OSVR/Private/OSVREntryPoint.cpp

```diff
@@ -29,13 +29,33 @@ DEFINE_LOG_CATEGORY(OSVREntryPointLog);
 
 OSVREntryPoint::OSVREntryPoint()
 {
+}
+
+void OSVREntryPoint::Initialize()
+{
+    // avoid double initialization
+    if (osvrClientContext != nullptr)
+    {
+        // too spammy to log here
+        return;
+    }
+
+    // hack: to avoid delay on OSVR server connection attempt when not using OSVR, don't initialize if nohmd command line param specified
+    if (FParse::Param(FCommandLine::Get(), TEXT("nohmd")))

```

Comment:
I just meant something like an argument to `Initialize` indicating whether it should attempt to connect if nohmd is set. In this case, OSVRInput would pass true, because it wants OSVR to connect even if the game is not running in VR mode. OSVR (hmd) would pass false.